packages/ui/components/form/timezone-select/TimezoneSelect.tsx

```diff
@@ -87,6 +87,7 @@ export function TimezoneSelect({
         menu: (state) =>
           classNames(
             "rounded-md bg-default text-sm leading-4 text-default mt-1 border border-subtle",
+            state.selectProps.menuIsOpen && "shadow-dropdown", // Add box-shadow when menu is open
```

Comment:
This is not related to this issue

Reply:
"I have submitted a pull request yesterday for this issue #8582, which has not yet been merged. That is why these changes are still in this pull request